### ISSUE TITLE
odd behavoir when duplicating nodes, or perhaps some bugs in fixtures / shapes right now

### ISSUE THREAD

**Issue by jordo** [briansemrau/godot_box2d#35]
https://youtu.be/K0108nf3W-4

**Comment by briansemrau** [comment]
just curious, are you adding shapes from a script, and if so what's the code?

**Comment by jordo** [comment]
just calling duplicate() and adding

**Comment by jordo** [comment]
```
var new_body = $Box2DPhysicsBody2.duplicate()
self.add_child(new_body)
```

**Comment by jordo** [comment]
lol, ya i have some weirdness going on right now haha

**Comment by briansemrau** [comment]
From the video, it looks like what I would expect

**Comment by jordo** [comment]
should be able to close with capsule fix in #36

**Comment by jordo** [comment]
fixed with #36
